src/components/atoms/OcCheckbox/OcCheckbox.vue

```diff
@@ -133,18 +133,17 @@ export default {
 
 .oc-checkbox {
   @include oc-form-check-size(1);
-
   -webkit-appearance: none;
   -moz-appearance: none;
-  background-color: var(--oc-color-input-bg);
-  background-position: 50% 50%;
-  background-repeat: no-repeat;
-  background-size: 80%;
-  border: 1px solid var(--oc-color-input-border);
-  border-radius: 3px;
+
+  background-position: 50% 50% !important;
+  background-repeat: no-repeat !important;
+  border: 2px solid var(--oc-color-input-border);
+  border-radius: 5px;
```

Comment:
you can extend the `.oc-rounded` class instead of setting a border-radius manually.